spec/persister/test_builder/persister_helper.rb

```diff
@@ -0,0 +1,35 @@
+class TestBuilder
+  module PersisterHelper
+    extend ActiveSupport::Concern
+
+    # builder_class for add_collection()
+    def cloud
+      ::TestBuilder::CloudManager
+    end
+
+    # builder_class for add_collection()
+    def network
+      ::TestBuilder::NetworkManager
+    end
+
+    # @param _extra_settings [Hash]
+    def builder_settings(_extra_settings = {})
+      opts = super
+      opts[:adv_settings] = options.try(:[], :inventory_collections).try(:to_hash) || {}
+
+      opts
+    end
+
+    # Returns list of target's ems_refs
+    # @return [Array<String>]
+    def references(collection)
+      target.try(:references, collection) || []
```

Comment:
moved to TargetCollection directly instead of 
https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/inventory/persister/builder/persister_helper.rb#L113